pages/roles.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Roles
```

Comment:
Per Slack convo – I think we should hold on this until we have a plan for creating this collaboratively with each chapter. Doing the design of this collaboratively means we'll be able to surface all kinds of good ideas from the folks who participate and also that the folks who fulfill each role on a cross-functional team will feel ownership in the new recommendations, rather than feeling as though they're having something pushed onto them.